### PR TITLE
feat: fix landing page based on design spec

### PR DESCRIPTION
**Pull request by linooohon**
## Types of changes
**Thanks for sending a pull request! Please fill in the following content to let us know better about this change.**
Please put an `x` in the box that applies

- [ ] **Bugfix**
- [x] **New feature**
- [ ] **Refactoring**
- [ ] **Breaking change** (any change that would cause existing functionality to not work as expected)
- [ ] **Documentation Update**
- [ ] **Other (please describe)**

## Description
**Describe what the change is**
- Fix landing page based on design spec.

## Steps to Test This Pull Request
1. Go to `/accounts/login`

## Expected behavior

與 Designer 討論過後，目前樣式更改為(Update at 2022/Apr/22 ):

Screen Shot 2022-04-22 at 5 30 49 PM

Screen Shot 2022-04-22 at 5 32 49 PM

Screen Shot 2022-04-22 at 5 33 36 PM

Screen Shot 2022-04-22 at 5 34 16 PM

## Related Issue

#1059

### CODE REVIEW

File: src/locale/zh_Hant/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -1565,6 +1565,20 @@ msgstr "審查"
 msgid "Log out"
 msgstr "登出"
 
+#: templates/default/_includes/nav/nav_base.html:180
+#: templates/default/_includes/nav/nav_base.html:181
+#: templates/default/_includes/nav/nav_base.html:189
+#: templates/default/_includes/nav/nav_base.html:190
+msgid "EN"
+msgstr "英文"
```
Comment (uranusjr) [inline src/locale/zh_Hant/LC_MESSAGES/django.po:1573]: 不確定這有沒有在 design 裡面，只寫個「英」「中」會不會比較好？

File: src/pycontw2016/settings/base.py [hunk 2]
```diff
@@ -163,12 +163,19 @@
...
+}
+
 if 'en-us' not in locale.LANG_INFO:
     locale.LANG_INFO['en-us'] = {
         'bidi': False,
         'code': 'en-us',
         'name': 'English (US)',
-        'name_local': 'English (US)',
+        'name_local': 'EN',
     }
```
Comment (uranusjr) [inline src/pycontw2016/settings/base.py:179]: 這兩個還有需要嗎？印象中 Django 的 language code 有改過，有人可以確認一下現在 `LANG_INFO` 裡有沒有這兩個值嗎？

Comment (uranusjr) [reply src/pycontw2016/settings/base.py:179]: 那好像可以拿掉 `zh-hant`?

File: src/locale/zh_Hant/LC_MESSAGES/django.po (hunk 1 above)
Comment (linooohon) [reply src/locale/zh_Hant/LC_MESSAGES/django.po:1573]: 這個跟 designer 討論過後，目前討論為 「英文」「中文」！

File: src/pycontw2016/settings/base.py (hunk 2 above)
Comment (linooohon) [reply src/pycontw2016/settings/base.py]: 感謝提供資訊，已拿掉。

**Comment by linooohon** [comment]
與 Designer 討論過後，版面做了一些更動，已更新截圖在 [Expected behavior](https://github.com/pycontw/pycon.tw/pull/1061#pullrequest-872431900), commit [c1a9d2f](https://github.com/pycontw/pycon.tw/pull/1061/commits/c1a9d2f9079dce1eb61fd339651ef24f7b673a55).

**Pull request merged**